R/tokenizer.R

```diff
@@ -34,7 +34,8 @@ NULL
 
 #' @export
 #' @rdname Tokenizers
-#' @param na String to use for missing values.
+#' @param na Character vector of strings to use for missing values. Set this
```

Comment:
Happy to make both changes. `utils::read.csv` has `na.strings = "NA"` by default:
```r
example = "a,b
1,foo
,
"
f = textConnection(example)
read.csv(f, colClasses=c(a="integer", b="character"))
#>    a   b
#> 1  1 foo
#> 2 NA    
close(f)
```
As you can see though, it turns empty values into `NA` where the column type is logical, integer or numeric, so I think extending the same behaviour to character columns by default is pretty reasonable.